GoogleForCreators/web-stories-wp
# Media: Use base color while loading in assets

## Context



## Summary

Is base color is present in the resource, use as background color

## Relevant Technical Choices

Use a new state for each element because it forces a re-render. 

## To-do



## User-facing changes


<img width="1792" alt="Screenshot 2021-11-22 at 11 49 07" src="https://user-images.githubusercontent.com/237508/142856842-067dc4bd-07b1-48eb-80a5-41524196a707.png">

## Testing Instructions






This PR can be tested by following these steps:

1. Open the story editor. 
2. See color background while video / images are loading. 
3. Search media / filter media. 
4. See color background while video / images are loading. 


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9647
